Hardware/Mainboard/SuperIOHardware.cs

```diff
@@ -353,6 +354,36 @@ internal sealed class SuperIOHardware : Hardware {
               for (int i = 1; i < superIO.Fans.Length; i++)
                 f.Add(new Fan("Fan #" + (i + 1), i));
               break;
+
+            case Model.ROG_ZENITH_EXTREME: // IT8665E
+              v.Add(new Voltage("CPU VCore", 0, 10, 10, 0, false));
+              v.Add(new Voltage("DRAM AB", 1, 10, 10, 0, false));
+              v.Add(new Voltage("+12V", 2, 5, 1,0, false));
+              v.Add(new Voltage("+5V", 3, 1.5f, 1, 0, false));
+              v.Add(new Voltage("SB 1.05V", 4, 10, 10, 0, false));
+              v.Add(new Voltage("DRAM CD", 5, 10, 10, 0, false));
+              v.Add(new Voltage("1.8V PLL", 6, 10, 10, 0, false));
+              v.Add(new Voltage("+3.3V", 7, 10, 10, 0, false));
+              v.Add(new Voltage("VBAT", 8, 10, 10, 0, false));
+
+              t.Add(new Temperature("CPU", 0));
+              t.Add(new Temperature("Mainboard", 1));
+              t.Add(new Temperature("CPU Socket", 2));
+              t.Add(new Temperature("Temperature #4", 3));
+              t.Add(new Temperature("Temperature #5", 4));
+              t.Add(new Temperature("VRM", 5));
+
+              f.Add(new Fan("CPU Fan", 0));
+              f.Add(new Fan("Chassis Fan 1", 1));
+              f.Add(new Fan("Chassis Fan 2", 2));
+              f.Add(new Fan("H AMP Fan", 3));

```

Comment:
What is an H AMP Fan? Any better name? 

Could you also make the casings and names in this file to the same one? I see VBAT, Battery and VBat for example. Also CPU Vcore, VCORE, VCore, etc. I think we should go for a little better readability when possible, so VBAT can be Vbat. VCORE can be Vcore (no need for CPU in the name).

Reply:
"H AMP Fan" is the name of the fan connector.
I ajust a VBAT variants to: Vbat

Reply:
I looked it up, it's the High Amperage Fan, I think it should be named as such, I don't think anyone would understand what it's for otherwise.

Reply:
In every location in the user manual the fan has the name "H AMP Fan".
https://dlcdnets.asus.com/pub/ASUS/mb/socketTR4/ROG_ZENITH_EXTREME/E13369_ROG_ZENITH_EXTREME_UM_V2_WEB.pdf
![fan1](https://user-images.githubusercontent.com/7898747/60511633-c1343480-9cd2-11e9-9e56-5db95b6a0a2f.PNG)
![fan2](https://user-images.githubusercontent.com/7898747/60511634-c1cccb00-9cd2-11e9-89b3-0dae06a5285c.PNG)

Reply:
Yeah, but also CHA_FAN, which is always Chassis Fan in project. I don't know why one would prefer abbreviations like this that don't make a lot of sense,